skuid/skuid-cli
# PLIN-4595: Fix request entity too large error

# Jira Issue URL
https://inflight.atlassian.net/browse/PLIN-4595


# High-Level Description
Added gzip compression to deploy payloads, specifically if filter values also have to be sent to pliny. Also added .vscode to .gitignore file to help with testing

# QA
- Make sure to review this in conjunction with https://github.com/skuid/pliny/pull/1984
- Download [this](https://github.com/skuid/skuid-cli/files/12097323/original-payload.zip) payload and Slack me to ask for an extra large file that you'll need to add to the payload to test that compression helps (I literally couldn't get it to attach to the JIRA ticket or here)
- You should be able to successfully deploy that payload to one of your local sites (on previous versions of CLI it should fail because of the Opening.mp4 file included in the files section)
